src/ZfcDatagrid/Renderer/PHPExcel/Renderer.php

```diff
@@ -103,14 +103,26 @@ public function execute()
                         break;
 
                     case 'ZfcDatagrid\Column\Type\DateTime':
-                        if ($value instanceof \DateTime) {
-                            $value->setTimezone(new \DateTimeZone($col->getType()
-                                ->getOutputTimezone()));
+                        /* @var $dateType \ZfcDatagrid\Column\Type\DateTime */
+                        $dateType = $col->getType();
+                        
+                        if (! $value instanceof \DateTime && ! is_object($value)) {
```

Comment:
maybe change `!is_object` to [is_scalar](http://php.net/manual/de/function.is-scalar.php), so check also that its no array and no resource